docker-compose.override.yml

```diff
@@ -51,6 +51,7 @@ services:
   minio:
     image: quay.io/minio/minio
     ports:
+      - 9000:9000
```

Comment:
This is not a big deal but `backend` already exposes port 9000, doesn't it? It should crash when you start both minio and backend through docker.

This would be the explanation for your problem: You started `backend` on your host and `minio` via docker.

Reply:
It does not on my end
Without it i get
<img width="545" height="131" alt="image" src="https://github.com/user-attachments/assets/5f6c9b43-844f-49ce-811f-fdb4e2212203" />

I check again

Reply:
Hm, this is annoying indeed. The tests might not setup the proxy that forwards the calls to minio. It's really difficult to find a solution that works for all scenarios here. Why don't the tests fail on CI? The backend server is constantly running?

Reply:
Maybe the solution is to not have calls to S3 in our integration tests? Just E2E tests where the backend will be running along with the proxy?

Reply:
I'd appreciate if you think about a proper solution. We actually have an working s3 local setup. Its fine to test with it. We just have trouble with the service exposure in different environments. Either via backend proxy, via locahost:port or dockeraddress:port.
A solution that works in all scenarios would be very good.